macosx/ArcemController.m

```diff
@@ -145,6 +148,7 @@ - (void)destroyEmulatorThread
  */
 - (void)awakeFromNib
 {
+    [super awakeFromNib];
     int i;
 
     // Create an initial emulator thread
```

Comment:
The call to `[[arcemView window] makeKeyAndOrderFront: self];` a few lines down from here needs to be on the main thread.

Reply:
From what I can tell, `-awakeFromNib` is called from the main thread.

Reply:
I can't seem to reproduce the issue any more, so it must have been a false positive on my part.